index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+  
+    <title>My-PhotoHub📸</title>
+   
+    <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
+    <link rel="stylesheet" href="./gallery.css">
+    <script type="module" src="./index.js"></script>
+
+  </head>
+
+  <body>
+
+  <!-- NAVIGATION BAR STARTS NAVIGATION BAR STARTS -->
+ 
+  <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+    <a class="navbar-brand" href="#">My-PhotoHub📸</a>
+</nav>
+
+    <p class="main">
```

Comment:
Is it possible to render the content of the `p` tag, i.e. the gallery itself center-aligned horizontally using bootstrap classes and/or custom CSS styles? Or use some other technique to fix the inconsistencies in the left and right margins.